metadata_service/proxy/atlas_proxy.py

```diff
@@ -344,6 +345,30 @@ def _serialize_columns(self, *, entity: EntityUniqueAttribute) -> \
             )
         return sorted(columns, key=lambda item: item.sort_order)
 
+    def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
+        reports = []
+        if guids:
+            report_entities_collection = self._driver.entity_bulk(guid=guids)
+            for report_entities in report_entities_collection:
```

Comment:
you can quickly expand with https://github.com/verdan/pyatlasclient/pull/2 (available in pyatlasclient 1.0.4)

Reply:
yep , i fixed it adbe31f